src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_transformers.py

```diff
@@ -46,3 +46,22 @@ def reservation_details_output(result):
 
 def transform_reservation_details_list_output(result):
     return [reservation_details_output(item) for item in result]
+
+
+def pricesheet_get_properties(result):
+    result.unit_of_measure = str(result.unit_of_measure)
+    result.included_quantity = str(result.included_quantity)
+    result.unit_price = str(result.unit_price)
+    if result.meter_details:
+        result.meter_details.total_included_quantity = str(result.meter_details.total_included_quantity)
+        result.meter_details.pretax_standard_rate = str(result.meter_details.pretax_standard_rate)
+    return result
+
+
+def pricesheet_get_output(result):
+    result.pricesheets = [pricesheet_get_properties(item) for item in result.pricesheets]
+    return result
+
+
+def transform_pricesheet_get_output(result):
+    return pricesheet_get_output(result)
```

Comment:
why not combine transform_pricesheet_get_output() and pricesheet_get_output()?

Reply:
Either way is fine here.